rmk/src/lib.rs

```diff
@@ -285,20 +287,27 @@ pub(crate) async fn run_keyboard<
 
     #[cfg(feature = "storage")]
     let storage_fut = storage.run();
+
+    #[cfg(feature = "controller")]
+    let mut wpm_controller = WpmController::new();
+
     match select4(
         communication_task,
         #[cfg(feature = "storage")]
         select(storage_fut, via_fut),
         #[cfg(not(feature = "storage"))]
         via_fut,
+        #[cfg(feature = "controller")]
```

Comment:
Now enabling `controller` automatically enables `wpm_controller`, is it right?

Reply:
In the future this could be optional when we start to work on configuration for controllers.